### PR TITLE
Updated 2FA Authy verify webhook with new signature verification

### PR DESCRIPTION
**Pull request by po5i**
Updated Python, Ruby, JavaScript and PHP snippets. Previously the Authy request had individual form parameters, now it comes with a JSON body, so it changed the way they have to be read and sorted.

@hortega @joliveros could you give me a hand with Java and C# snippets when you can, please?

### CODE REVIEW

File: two-factor-authentication/verify-webhook/verify-webhook.5.x.rb [hunk 1]
```diff
@@ -8,16 +8,9 @@ class CallbackVerifier
...
-      .to_a
-      .concat(request.request_parameters.to_a)
-      .sort
-      .map{ |p| p.join('=')}
-      .join('&')
-
-    parameter_string = URI.encode(parameter_string)
+    # Sort and join the parameters on Rails
+    json_params = JSON.parse(request.body.read)
+    parameter_string = json_params.to_query
```
Comment (hortega) [inline two-factor-authentication/verify-webhook/verify-webhook.5.x.rb]: Are the parameters in the string sorted in alphabetical order?

Comment (po5i) [reply two-factor-authentication/verify-webhook/verify-webhook.5.x.rb]: Good catch. I don't see the sort anywhere but it "magically" worked when I tested it with the Authy JSON body.
I will review it more in detail.

Comment (po5i) [reply two-factor-authentication/verify-webhook/verify-webhook.5.x.rb]: It looks like on Rails `Hash#to_query` extension [sorts the query parameters](https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/object/to_query.rb#L82) before joining them.

Comment (hortega) [reply two-factor-authentication/verify-webhook/verify-webhook.5.x.rb]: Nice one! Gotta love magic :slightly_smiling_face:

**Comment by coderkat** [comment]
👍

**Pull request merged**